docs/migrate-module.md

```diff
@@ -0,0 +1,94 @@
+# Update an older module
+
+Module repos created before July 2022 don't include the logic that is needed to pass the continuous integration (CI) tests and to publish the modules to the IBM Cloud catalog. Follow these steps to update an existing repo in the terraform-ibm-modules GitHub organization.
+
+## Before you begin
+
+- If you're running Microsoft Windows, set up Windows Subsystem for Linux [WSL](https://ubuntu.com/wsl) and run commands within WSL.
```

Comment:
Perhaps we can just link to existing local dev setup to avoid duplicating content

Reply:
Generally like to prevent people from jumping around as they're getting set up, but this is kind of a one-off, so people will probably be ok with that here.